### DISCUSSION TITLE
stoicism/descendant/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#34] [General]
# stoicism/descendant/

https://shanenull.com/stoicism/descendant/

**Comment by shane0** [comment]
# more stories for descendents

- add work some of it will have to be private because I've worked at all sorts of places
- worked with every kind of medical office mainly home health and heart surgeons 
- healthcare IT 
- remote computer security software wow weird stories there
- broadcast television station 
- traveled to every kind of office law government fbi prisons and boeing

**Reply by shane0** [reply]
if i can't find the watercolor from 6yrs old i can recreate it lol i can see it in my memories

**Reply by shane0** [reply]
- make a video with stories about me & john irvine & ares of the map including the junkyard parks how far we'd bicycle or skateboard & the bridge over the river we jumped off + the stadium seats we jumped off w/ a high jump mat under it

**Comment by shane0** [comment]
# computations

Meditation
https://github.com/shane0/buddhism/issues/127#issuecomment-2558631021
